duckduckgo/Android
# Add disclaimer in ATP breakage reporting

Task/Issue URL: https://app.asana.com/0/414730916066338/1201487831849930/f

### Description
This task adds a disclaimer and an associated checkbox right when the user is about to submit an ATP breakage report.
Copy task can be found [here](https://app.asana.com/0/38424471409662/1201487899123451/f)

We have also removed some pixels that we were sending in the "Manage app protection" and added a link in the Settings page to our privacy policy

### Steps to test this PR
1. fresn install from this branch
1. go to settings -> privacy policy
1. VERIFY the DDG privacy policy page is displayed
1. enable ATP
1. go to "App Tracking Protection" -> "Manage protection for your apps"
1. disable protection for one of the apps
1. select "I'm ok being tracked in this app" and submit
1. VERIFY no pixel is send
1. disable protection for another app
1. select "For other reason" and submit
1. VERIFY no pixel is send
1. disable protection for another app
1. select "It stopped working as expected" and submit
1. VERIFY none of the pixels sent carries (broken) app information
1. tap "Next"
1. VERIFY none of the pixels sent carries (broken) app information
1. VERIFY the submit button is disabled
1. select one of the options in the radio buttons
1. VERIFY the submit button is disabled
1. check the checkbox
1. VERIFY the submit button is enabled
1. uncheck the checkbox
1. VERIFY the submit button is disabled
1. check the checkbox and submit
1. VERIFY the `m_atp_breakage_report` is sent (other impression/event pixels may also be sent)
